charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -28,6 +28,9 @@ metadata:
   labels:
     {{- include "pulsar.standardLabels" . | nindent 4 }}
     component: {{ .Values.bookkeeper.component }}
+    {{- with .Values.bookkeeper.labels }}
+    {{ toYaml . | indent 4 }}
```

Comment:
I think we shouldn't indent the line here, the indent function already did the work

Reply:
Thanks, fixed